masonry/src/contexts.rs

```diff
@@ -1069,6 +1074,16 @@ impl_context_method!(LayoutCtx<'_>, PaintCtx<'_>, {
     }
 });
 
+impl PaintCtx<'_> {
+    /// Whether debug paint is enabled.
+    ///
+    /// Debug paint can be enabled using the environment variable `MASONRY_DEBUG_PAINT`.
+    /// When enabled widget boundaries are outlined.
+    pub fn debug_paint(&self) -> bool {
```

Comment:
I'd like to brainstorm a different name here - to me, `debug_paint` sounds like an "action" rather than a property read. That is, this method name suggests that it performs some kind of debug operation itself.

This needn't block this from landing.